client/src/components/ImageCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class='card'>
+    <span class='asset'>
+      <img :src='require(`@/assets/media/${content.image ?? defaultImage}`)'
+           alt=''>
+    </span>
+    <div class='header-1'>{{ content.header }}</div>
```

Comment:
nit: It's more semantic to use `h1` and `h2` instead of `div`s with classs names here.

Reply:
good point - I initially had these then realized they were messing up my styling, but just added more styling to normalize. Thanks!